integration-tests/src/test/java/org/apache/druid/tests/query/ITQueryRetryTestOnMissingSegments.java

```diff
@@ -52,12 +51,11 @@
  * This class tests the query retry on missing segments. A segment can be missing in a historical during a query if
  * the historical drops the segment after the broker issues the query to the historical. To mimic this case, this
  * test spawns a historical modified for testing. This historical announces all segments assigned, but doesn't serve
- * all of them always. Instead, it can report missing segments for some
- * segments. See {@link ServerManagerForQueryErrorTest} for more details.
+ * all of them always. Instead, it can report missing segments for some segments.
  * <p>
  * To run this test properly, the test group must be specified as {@link TestNGGroup#QUERY_RETRY}.
  */
-@Test(groups = TestNGGroup.QUERY_RETRY)
+@Test(groups = TestNGGroup.QUERY_RETRY, enabled = false) // TODO: Temporarily disabled due to migration to embedded suite.
 @Guice(moduleFactory = DruidTestModuleFactory.class)
 public class ITQueryRetryTestOnMissingSegments
```

Comment:
@Fly-Style , is this test not being migrated in this PR?

Reply:
Nope, I think I wrote somewhere that it will be a part of the next consequent PR, if it is fine.
If not, I will expand the scope of this PR then.

Reply:
Thanks!